pkgs/development/python-modules/cryptolyzer/default.nix

```diff
@@ -56,11 +57,14 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "cryptolyzer" ];
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { };
```

Comment:
An explicit update script shouldn't be needed now that https://github.com/ngi-nix/ngipkgs/issues/1326 is resolved.